project/build.scala

```diff
@@ -206,7 +207,10 @@ object build extends Build {
         case ex: Exception if ex.getMessage.contains(nothingToCommit) => println(nothingToCommit)
       }
     },
+    // TODO: doesn't work at the moment, see https://github.com/sbt/sbt-pgp/issues/42
+    publishSigned := publish.value,
```

Comment:
More precisely, I can't disable `publish-signed` for the `readme` project. I do not want to publish it, just to use it to generate the website via Scalatex. However, `sbt-pgp` insists that I configure a publishing repository for `readme`, otherwise it errors out when I do `sbt publish-signed`.